install.sh

```diff
@@ -1,5 +1,10 @@
 #! /usr/bin/env bash
 
+if [[ "${BASH_VERSION%%.*}" -eq 3 ]]; then
+    echo "Atuin doesn't work on bash 3.2. Please upgrade your version of bash (possibly via homebrew or ports) and try again"
+	exit
```

Comment:
It is still good to notify users that Atuin doesn't offer full support for Bash 3.2.

```suggestion
    echo "Atuin has limited support for Bash 3.2. The config enter_accept cannot be turned off." >&2
    echo "To turn off enter_accept, please upgrade your version of bash (possibly via homebrew or ports)" >&2
```